resources/images/filters.go

```diff
@@ -174,6 +177,54 @@ func (*Filters) Padding(args ...any) gift.Filter {
 	}
 }
 
+// Dither creates a filter that dithers an image.
+func (*Filters) Dither(options ...any) gift.Filter {
+	ditherOptions := struct {
+		Colors     []string
+		Method     string
+		Serpentine bool
+		Strength   float32
+	}{
+		Colors:     []string{"000000ff", "ffffffff"},
+		Method:     "floydsteinberg",
+		Serpentine: true,
+		Strength:   1.0,
+	}
+
+	err := mapstructure.WeakDecode(options[0], &ditherOptions)
```

Comment:
Add a check for empty options slice and we should be all good.

Reply:
I guess empty options should give the defaults.

Reply:
I tested for an empty map, but not for zero args. Thanks.